### ISSUE TITLE
Problem with K8s 1.17

### ISSUE THREAD

**Issue by ksalihcavusoglu** [komljen/helm-charts#31]
"helm install --name efk --namespace logging akomljen-charts/efk" not working on k8s 1.17.

Error: unable to build kubernetes objects from release manifest: [unable to recognize "": no matches for kind "DaemonSet" in version "extensions/v1beta1", unable to recognize "": no matches for kind "Deployment" in version "apps/v1beta1"]

**Comment by ASangave** [comment]
@ksalihcavusoglu Were you able to resolve this issue? Because I am getting the same error for k8s 1.16.10

Error: validation failed: [unable to recognize "": no matches for kind "DaemonSet" in version "extensions/v1beta1", unable to recognize "": no matches for kind "Deployment" in version "apps/v1beta1"]

**Comment by ksalihcavusoglu** [comment]
@ASangave no, its about kubernetes version. We have to change it to app/v1 but i didnt have time to change chart. So i installed efk seperately and now working

**Comment by ASangave** [comment]
Yup. It worked when I downgraded my kubernetes version to 1.15
Thanks
